src/components/MuiVirtualizedTable/MuiVirtualizedTable.js

```diff
@@ -329,83 +587,118 @@ class MuiVirtualizedTable extends React.PureComponent {
+                        classes.noClick,
+                        classes.header
+                    )}
+                    variant="head"
+                    style={{ height: this.state.headerHeight }}
+                    align={
+                        columns[columnIndex].numeric || false ? 'right' : 'left'
+                    }
+                    ref={(e) => this._registerObserver(e)}
+                >
+                    {this.props.sortable
+                        ? this.sortableHeader({
+                              ...headerProps,
+                              columnIndex,
+                              key: { dataKey },
+                          })
+                        : this.simpleHeaderRenderer({
+                              ...headerProps,
+                          })}
+                </TableCell>
+            );
+        };
     }
 
-    componentWillUnmount() {
-        window.removeEventListener('resize', this._computeHeaderSize);
-        this.observer.disconnect();
-    }
+    makeSizedTable = (height, width, sizes, reorderedIndex, rowGetter) => {
+        const { sort, ...otherProps } = this.props;
+
+        return (
+            <Table
+                {...otherProps}
+                height={height}
+                width={width}
+                rowHeight={otherProps.rowHeight}
+                gridStyle={{ direction: 'inherit' }}
+                headerHeight={this.state.headerHeight}
+                className={otherProps.classes.table}
+                onRowClick={
+                    this.props.onRowClick &&
+                    /* The {...otherProps} just above would hold the slot onRowClick */
+                    this.onClickableRowClick
+                }
+                rowCount={reorderedIndex?.length ?? otherProps.rows.length}
+                rowClassName={({ index }) =>
+                    this.getRowClassName({ index, rowGetter })
+                }
+                rowGetter={({ index }) => rowGetter(index)}
+            >
+                {otherProps.columns.map(({ dataKey, ...other }, index) => {
+                    return (
+                        <Column
+                            key={dataKey}
+                            headerRenderer={this.makeHeaderRenderer(
+                                dataKey,
+                                index
+                            )}
+                            className={otherProps.classes.flexContainer}
+                            cellRenderer={this.cellRenderer}
+                            dataKey={dataKey}
+                            flexGrow={1}
+                            width={sizes[dataKey]}
+                            {...other}
+                        />
+                    );
+                })}
+            </Table>
+        );
+    };
 
     getCSVFilename = () => {
-        if (this.props.name && this.props.name.length > 0) {
+        if (!this.props.name && this.props.name.length > 0) {
```

Comment:
why ? maybe you means:
`if (this.props.name?.length > 0) {`

Reply:
Uh, half reversed hack for testing the else part :fearful:  Thnaks for noticing it